app/views/dormant_user_mailer/email_dormant_user.text.erb

```diff
@@ -0,0 +1,21 @@
+Dear <%= @user.display_name %>,
+
+Our projects still need you!
+
+Whether it is helping to classify galaxies, map organutan nesting sites, identify antibiotic resistant bacteria or to transcribe anti-slavery manuscripts, there are many exciting research projects to get involved with in the Zooniverse.
+
+Pick up where you last left off on [project name, with embedded link]<%= @last_project %>, or check out all our projects here: (https://zooniverse.org/#/projects)
```

Comment:
use the `@last_project` to get the `display_name` and slug attributes, don't print the object id.